source/voice_control/voice_remote_local_assistant.markdown

```diff
@@ -26,6 +26,7 @@ For the quickest way to get your local Assist pipeline started, follow these ste
    - Once the add-ons are started, head over to the integrations under {% my integrations title="**Settings** > **Devices & Services**" %}.
      - You should now see Piper and Whisper being discovered by the [Wyoming integration](/integrations/wyoming/).
        ![Whisper and Piper integrations](/images/assist/piper-whisper-install-new-02.png)
+     - If they aren't being discovered, you may need to reboot your system.
```

Comment:
That sounds wrong and should be reported as an issue instead

Reply:
It is reported as an issue since sep 2023. #28770

Reply:
That is something different.
The host fields aren't relevant for add-on usage, if they are: it is a bug.

The issue you reference is documentation feedback. If discovery doesn't work, that would be a bug, and should be raised as such in the core repository issue tracker.